<launch>


<arg name="roof_width"/>
<arg name="roof_height"/>
<arg name="inchworm_count"/>
<arg name="rate" default="10"/>
<node pkg="inchworm_algo" type="algo_node.py" name="algo_node" args="$(arg roof_width) $(arg roof_height) $(arg inchworm_count) $(arg rate)" output="screen"/>

<node pkg="inchworm_algo" type="algo_visualizer.py" name="algo_viz" args="$(arg roof_width) $(arg roof_height) $(arg rate)" output="screen"/>


</launch>